packages/react-components/src/components/FloatingPortal/types.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export interface FloatingPortalProps {
```

Comment:
Maybe it would be better to use the original FloatingPortalProps here?

Reply:
floating-ui doesn't export them